v-next/hardhat-ignition-viem/package.json

```diff
@@ -21,8 +21,8 @@
   "main": "dist/src/index.js",
   "types": "dist/src/index.d.ts",
   "scripts": {
-    "prebuild": "(tsc --build || true) && ts-node --project tsconfig.json ./scripts/compile-test-fixture-project.ts create2 && ts-node --project tsconfig.json ./scripts/compile-test-fixture-project.ts minimal",
-    "build": "tsc --build",
+    "prebuild": "node ./scripts/compile-test-fixture-project.js create2 && node ./scripts/compile-test-fixture-project.js minimal",
```

Comment:
We should look at this again after the initial public alpha.
We do a prebuild to generate the Viem types for the contracts in the fixture projects.
In tests we then have type assertions, so the build will fail of the Viem types are not what we expect.
I think we want those type level checks as they are so integral to the plugin.
However I am less keen with the two step build shenanigans.